### PR TITLE
Mars 2020 Mission Dictionary Release 1.0.0.0, first public release

### PR DESCRIPTION
**Pull request by sslavney**
## Summary

First public release of the Mars 2020 Mission Dictionary

### CODE REVIEW

**Comment by sslavney** [comment]
Lyle Huber at the Atmospheres Node ought to be a reviewer but his name is not in the list. I'll notify him by email.

**Comment by jordanpadams** [comment]
@sslavney I invited @lylehuber to the organization so he can be added as a reviewer.

I also created a new team you can manage for the Mars2020 LDD: https://github.com/orgs/pds-data-dictionaries/teams/mars2020-ldd-stewardship-team .

Additionally, I updated the [CODEOWNERS](https://github.com/pds-data-dictionaries/ldd-mars2020/blob/main/.github/CODEOWNERS) for your repo to that team, and the settings on the repo so that the team will always be notified their review is requested when a pull request is created:

Screen Shot 2021-07-17 at 11 13 29 AM

File: Users-Guide.md [hunk 1]
```diff
@@ -1,96 +1,106 @@
...
+The namespace for the Mars 2020 Mission Dictionary is http://pds.nasa.gov/pds4/mission/mars2020/v1, abbreviated "mars2020:".
+
+# Dictionary Structure
+
+An outline of the classes and attributes in the dictionary is given below.
+
+```
+    <mars2020:Mars2020_Parameters>
+       <mars2020:Observation_Information>  
+          <mars2020:release_number>
```
Comment (myche) [inline Users-Guide.md:59]: Again, I'm hoping we can eventually move to using an _Archive_ LDD that could contain things like `release_number` (as well as other items).  I'd love for GEO and IMG to work together on such a thing.  But, until then, this is cool.

File: Users-Guide.md [hunk 2]
```diff
@@ -1,96 +1,106 @@
...
+          [ <mars2020:start_sol_number>  
+            <mars2020:stop_sol_number>  ]
+          <mars2020:start_local_mean_solar_time> 
+          <mars2020:stop_local_mean_solar_time> 
+          <mars2020:start_local_true_solar_time> 
+          <mars2020:start_local_true_solar_time_sol> 
+          <mars2020:stop_local_true_solar_time> 
+          <mars2020:stop_local_true_solar_time_sol> 
+          <mars2020:start_solar_longitude> 
+          <mars2020:stop_solar_longitude>
```
Comment (myche) [inline Users-Guide.md:76]: These seem to be more generic than just M2020.  Are there some use cases where M2020 is _different_ than other rovers that are being captured here?  This is fine, if so.  I'm just ignorant of the context.

**Review by myche — APPROVED** [review]
We on the road for 1.0!

(Thanks for pushing in the `sinetable` change for us!)

**Pull request merged**